### DISCUSSION TITLE
No dynamic path possible

### DISCUSSION THREAD

**Discussion by SaschaValentin** [webstudio-is/webstudio-community#157] [General]
I´ve tried to add a dynamic path to a website I´ve build using Webflow but that seems to be impossible.

In contrast to the tutorial videos on YouTube, unfortunately you cannot create dynamic paths - only in the Pro version.

Hence my question: When am I in the open source version? Only if I host Webstudio myself? But that's not possible at the moment, if I understood this post correctly: https://github.com/webstudio-is/webstudio-community/discussions/150? Is that right?

**Comment by kof** [comment]
@SaschaValentin hi, yes, dynamic pages are available on pro or self-hosted.

**Reply by kof** [reply]
self-hosting the builder itself is currently non-trivial, so yes, best option for now is to use pro, though its not intentional, we just didn't have time to make a self-hosted version easier to install due to pressure on features side.

**Reply by kof** [reply]
I mean there is a way to self-host, its just not a 1-click thing. Beyond that you can def. self-host the site you produced, even with dynamic data. This is relatively easy with cli. But to build it you still need access to pro plan if you are building on webstudio cloud

**Reply by SaschaValentin** [reply]
Thanks for your reply

**Reply by kof** [reply]
Oh btw you can use our devcontainer that we use for development to use the builder locally with ease, then self-host the produced site using anything, since the two are decoupled. So actually if you at least know how to use docker, you can get it all working relatively easily.
